### ISSUE TITLE
Swagger Object-level Parameters Raise an Error

### ISSUE THREAD

**Issue by hunterji** [Yelp/swagger_spec_validator#29]
I'm using swagger object-level parameters with swagger 2.0. When I reference them in the path's parameters property as "$ref": "#/parameters/MyParam" I get an error:
```swagger_spec_validator.common.SwaggerValidationError: 'name'```

As an example the instagram.yaml demo file used in http://editor.swagger.io raises the same error for the "user-id" and "tag-name" parameters.

Both my swagger.json and the demo file mentioned render in Swagger-UI without error.

**Comment by analogue** [comment]
Got an example of the schema and a way to reproduce the issue?

**Comment by hunterji** [comment]
The instagram demo mentioned above is available [here](https://github.com/swagger-api/swagger-editor/blob/master/app/spec-files/instagram.yaml) as yaml (I was going to attach, but can't and I'm not going to paste 1200+ lines).

 I'm just using using this bit of python pointed at the file hosted locally to generate the error:
```python
from swagger_spec_validator import validate_spec_url
validate_spec_url('http://127.0.0.1:8080/swagger.json')
```

----------------------

**Comment by analogue** [comment]
You can pick up the fix in forthcoming 1.0.12.
